jobs/ci-kubernetes-e2e-gke-1.4-latest-cvm-kubectl-skew.env

```diff
@@ -8,3 +8,6 @@ KUBE_GKE_IMAGE_TYPE=container_vm
 ZONE=us-central1-c
 
 KUBEKINS_TIMEOUT=120m
+
+# Enable when testing upgrades from a version < 1.6 to a version >= 1.6
+ENABLE_LEGACY_ABAC=true
```

Comment:
ah, ok. I'll let @cjcullen update the non-upgrade gke jobs with `--enable-legacy-abac=false` gcloud flag at some point in the future, then. should I remove this from all the gke env files to avoid confusion?